packages/genesys-spark-components/src/components/stable/gux-accordion/tests/gux-accordion.e2e.playwright.ts

```diff
@@ -1,27 +1,48 @@
-import { expect } from '@playwright/test';
-import { test } from '@stencil/playwright';
-import AxeBuilder from '@axe-core/playwright';
+import { analyze, setContent, test } from '@test/playwrightTestUtils';
 
 test.describe('gux-accordion', () => {
-  test('should render as expected', async ({ page }) => {
-    // The path here is the path to the www output relative to the dev server root directory
-    await page.goto(
-      '/components/stable/gux-accordion/tests/gux-accordion.e2e.playwright.html'
+  test('renders', async ({ page }) => {
```

Comment:
Do we need to add command in `package.json` to run the playwright tests.

Reply:
There is one at the moment but it is not hooked up to the other tests yet. I will do that in this issue https://inindca.atlassian.net/browse/COMUI-3596.